cmd/pod-scaler/frontend/src/app/utils/useDocumentTitle.ts

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+// a custom hook for setting the page title
+export function useDocumentTitle(title: string) {
```

Comment:
https://github.com/nfl/react-helmet is a nice utility for this

Reply:
This was taken from the React PatternFly Seed repo verbatim, will open a PR there to update!